rules-tests/DowngradePhp74/Rector/Array_/DowngradeArraySpreadRector/Fixture/with_iterable_pseudotype_array.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
File name must have `.inc` suffix, on this case: `with_iterable_pseudotype_array.php.inc`, also, please add namespace `namespace Rector\Tests\DowngradePhp74\Rector\Array_\DowngradeArraySpreadRector\Fixture;`